_includes/story.md

```diff
@@ -882,11 +899,11 @@ end
 
 In the code above the date range would be delimited by having the earliest timestamp of the date range as the `minscore` and the latest timestamp as the `maxscore`.
 
-However, we ran into concurrency issues because we had to perform 3 separate operations, for which we could not guarantee the atomicity: getting the count of jobs from Redis, removing the count from Redis, incrementing the count in our code and pushing the count back in Redis.
+However, we ran into concurrency issues because we had to perform 3 separate operations, for which we could not guarantee the atomicity of this whole transaction: getting the count of jobs from Redis, removing the count from Redis and incrementing the count in our code and pushing the count back in Redis.
```

Comment:
I prefer the comma over the "and". It's a better division of the three steps imo, now it reads a bit weird with two "and" chained.